automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -314,7 +315,11 @@ trait AzureBillingBeforeAndAfter extends FixtureAnyFreeSpecLike with BeforeAndAf
 
     try
       testCode(response)
-    finally
+    catch {
+      case e: Throwable =>
+        println(s"Exception occurred during test: ${e}")
```

Comment:
Just a question, do you have a rule of thumb about printing vs. logging? I usually log, and only do some println for debugging, so I am wondering if I am a bit off when I code in leonardo now :)

Reply:
The logger is usually better, but its not in scope here, so I threw this in here. I should probably be using the logger though tbh, but I'm not really sure of the benefit